openrlhf/trainer/ppo_trainer.py

```diff
@@ -476,7 +476,7 @@ def training_step_critic(self, experience: Experience) -> Dict[str, float]:
         }
         return status
 
-    def evaluate(self, dataloader, global_step):
+    def evaluate(self, args, dataloader, global_step):
```

Comment:
maybe only pass args.extra_rm_args instead of the full thing for better modularization and consistency with training part?

Reply:
done in 0c3463e7521f669df5571476ea08bf9330a6731c